static/app/components/workflowEngine/gridCell/userCell.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled';
+
+import UserBadge from 'sentry/components/idBadge/userBadge';
+import {IconSentry} from 'sentry/icons';
+import type {AvatarUser} from 'sentry/types/user';
+
+export type UserCellProps = {
+  user: 'sentry' | AvatarUser;
+  className?: string;
+  disabled?: boolean;
+};
+
+export function UserCell({user, disabled = false, className}: UserCellProps) {
```

Comment:
should there be a tooltip here with the user's email or name? not sure if this is the current expected behavior when you hover 😅 

<img width="96" alt="Screenshot 2025-04-07 at 4 02 53 PM" src="https://github.com/user-attachments/assets/be9afa40-47ad-4500-b094-fbc62fe93e47" />